.mega-linter.yml

```diff
@@ -0,0 +1,23 @@
+# Configuration file for MegaLinter
+# See all available variables at https://megalinter.io/configuration/ and in linters documentation
+---
+EXCLUDED_DIRECTORIES:
+  - test/fixtures/
+
+DISABLE_LINTERS:
+  - REPOSITORY_KICS
+  - GO_GOLANGCI_LINT
+  - GO_REVIVE
+  - REPOSITORY_GRYPE
+  - SPELL_LYCHEE
```

Comment:
These deactivations need a comment

Why are they deactivated?

You provided the information in the PR description, you could add the information here